File: app/src/main/java/com/testeandroidv2/repository/response/Statement.java
package com.testeandroidv2.repository.response;

import com.google.gson.annotations.SerializedName;

public class Statement {

    @SerializedName("title")
    private String title;

    @SerializedName("desc")
    private String description;

    @SerializedName("date")
    private String date;

    @SerializedName("value")
    private String value;

    public String getTitle() {
        return title;
    }

    public String getDescription() {
        return description;
    }

    public String getDate() {
        return date;
    }

    public String getValue() {
        return value;
    }
}

File: app/src/main/java/com/example/projetobank/data/model/Statement.kt
package com.example.projetobank.data.model

import com.google.gson.annotations.SerializedName

data class Statement(
    @SerializedName("date")
    val date: String,
    @SerializedName("desc")
    val desc: String,
    @SerializedName("title")
    val title: String,
    @SerializedName("value")
    val value: Double
)